### PR TITLE
Add support for tag_cloud

### PR DESCRIPTION
**Pull request by radoeka**
This pull request add supports for the tag_cloud plugin.

There are two new strings added, that could be translated.
I would like to do this, but for this the new strings are to be added to the respective language file.
Can you that, please.

### CODE REVIEW

File: static/stylesheet/tag_cloud.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+ul.tagcloud {
```
Comment (alexandrevicenzi) [inline static/stylesheet/tag_cloud.css:1]: This code should be added to style.less

File: templates/tags.html [hunk 2]
```diff
@@ -2,12 +2,33 @@
 
 {% block title %} &ndash; {{ _('Tags') }}{% endblock %}
 
+{% block meta %}
```
Comment (alexandrevicenzi) [inline templates/tags.html:5]: This is not needed if the code is in style.less

File: templates/tags.html [hunk 3]
```diff
@@ -2,12 +2,33 @@
...
+  {% endif %}
+{% endblock %}
+
 {% block content %}
 <article class="single">
   <header>
     <h1 id="tags">{{ _('Tags') }}</h1>
   </header>
-  <div>
+
```
Comment (alexandrevicenzi) [inline templates/tags.html:16]: This does not make sense in here, this template is for a list of tags, tag cloud is something for home pages.

**Comment by alexandrevicenzi** [comment]
To be honest, tag cloud makes no sense to me in the place where it's being put.

Thinking a bit more, I see no place for a tag cloud in the theme, it does not fit anywhere.
Some sites provide tag clouds on sidebars or footers, but it just does not look good in such places.

**Comment by alexandrevicenzi** [comment]
Looking at your website, it may look better if done this way:

image

As this is a plugin that needs to be enabled, it could be added if it looks like this.
But as I said, it does not make much sense, since the page already has a list of tags and the article count.

**Comment by radoeka** [comment]
I like it, because it shows which tag its used most often.

But, when you state it does not make sense; does it mean it will not be added at all?  Or is it still possible to integrate it, as it can be disabled?

When the css code is put in style.less, how is converted to style.min.css so it can be tested?  I did not (don't) want to add it to overall stylesheet as it is only needed when tag_cloud is enabled and only in tags.html.

**Comment by radoeka** [comment]
Alexandre, the change is now as you advised.  Can you accept the pull request?

**Comment by alexandrevicenzi** [comment]
> Alexandre, the change is now as you advised. Can you accept the pull request?

There are still things to be addressed, the main one is to keep styles in one single place.

Once you fix that, I'll merge it.

**Comment by radoeka** [comment]
Can you provide or refer to the instructions how to convert the style.less to style.min.css?
Or can you provide the the css code for style.less?

**Comment by alexandrevicenzi** [comment]
Just paste your CSS code into the LESS file, it will work. There~s no need for another CSS file.

For style.min.css you need to have NodeJS, npm, and a bunch of dependencies, such as Gulp.
Might be too hard for you, if you push the changes in the LESS file I can update the minified version for you.

**Comment by radoeka** [comment]
Hi, thanks for bringing this up.  I didn't have time to make the PR for the diff including the 'less' code.  I remember that I had experimented with the less code, and that it worked around mid August.  Using a less to minified online decoder, that worked well.
But as stated at that point it stalled.  I'm still using pelican with flex.  I don't have too much time to look into this, perhaps around xmas again (although there aren't a lot of holidays during that time, not as how it is normally).

**Comment by alexandrevicenzi** [comment]
I'm going to close this PR and work on tag_cloud support for Flex 3.0.

**Pull request closed without merging**